index.html

```diff
@@ -127,7 +128,259 @@ <h2>Multi-Post Stories</h2>
                                 </ul>
                             </li>
                         </ul>
-                        <button type="button">See Project</button>
+                        <button type="button" class="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="myProjects">
```

Comment:
- [ ] As requested in the project requirements it is mandatory that you use CSS Grid, please fix this :+1:  
It's important that you show that you are able to use CSS Grid as this is an important and useful tool for working on the layout of UI elements
![image](https://user-images.githubusercontent.com/28109626/148609236-956e368c-2b88-4dd9-beb6-2a22ee50183a.png)

Reply:
Hello @mike2611 I have fixed that and have used CSS grid in at least one place in my design.